prototype_source/quantization_2_0_tutotial.rst

```diff
@@ -0,0 +1,567 @@
+
+    # Step 2: Insert observers or fake quantize modules
+    quantizer = qq.QNNPackQuantizer()
+    operator_config = qq.get_symmetric_quantization_config(is_per_channel=True)
+    quantizer.set_global(operator_config)
+    prepared_graph_module = prepare_pt2e(exported_graph_module, quantizer)
+
+    # Step 3: Quantize the model
+    convered_graph_module = convert_pt2e(prepared_graph_module)
+
+    # Step 4: Lower Reference Quantized Model into the backend
+
+Inside the Quantizer, we will use the `QuantizationAnnotation API <https://docs.google.com/document/d/1tjIsL7-uVgm_1bv_kUK7iovP6G1D5zcbzwEcmYEG2Js/edit#>`__
+to convey user's intent for what quantization spec to use and how to
+observe certain tensor values in the prepare step. Now, we will have a step by step
+tutorial for how to use the `QuantizationAnnotation API` to create a quantizer.
+
+1. Define QuantizationConfig
+--------------------------------------------------------
+
+QuantizationConfig defines the data type and qscheme for activation, weight and bias. Suppose
+we want to define:
+
+-  Activation: `int8` data type, `per_tensor_affine` quantization, `HistogramObserver`
+-  Weight    : `int8` data type, `per_channel_symmetric` quantization, `PerChannelMinMaxObserver`
+-  Bias      : `float` data type, `PlaceholderObserver`
+
+::
+
+    def get_symmetric_quantization_config():
+        act_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = \
+            HistogramObserver
+        act_quantization_spec = QuantizationSpec(
+            dtype=torch.int8,
+            quant_min=-128,
+            quant_max=127,
+            qscheme=torch.per_tensor_affine,
+            is_dynamic=False,
+            observer_or_fake_quant_ctr=act_observer_or_fake_quant_ctr.with_args(eps=2**-12),
+        )
+
+        weight_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = PerChannelMinMaxObserver
+        extra_args: Dict[str, Any] = {"eps": 2**-12}
+        weight_quantization_spec = QuantizationSpec(
+            dtype=torch.int8,
+            quant_min=-127,
+            quant_max=127,
+            qscheme=torch.per_channel_symmetric,
+            ch_axis=0,
+            is_dynamic=False,
+            observer_or_fake_quant_ctr=weight_observer_or_fake_quant_ctr.with_args(**extra_args),
+        )
+
+        bias_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = PlaceholderObserver
+        bias_quantization_spec = QuantizationSpec(
+            dtype=torch.float,
+            observer_or_fake_quant_ctr=bias_observer_or_fake_quant_ctr
+        )
+        quantization_config = QuantizationConfig(
+            act_quantization_spec, weight_quantization_spec, bias_quantization_spec, is_qat
+        )
+        return quantization_config
+
+2. Define the BackendQuantizer
+--------------------------------------------------------
+
+Then we will define the skeleton of a BackendQuantizer. The annotatation methods for each operation will be
+defined later.
```

Comment:
Suggest to explain the semantics of each method of the quantizer a backend needs to implement.

Reply:
Thanks for the comment. Add docstring for each method.